assets/scss/main.css.scss

```diff
@@ -32,3 +32,9 @@
 .reversefootnote {
   @include u-font-family("mono");
 }
+
+.usa-layout-docs__sidenav {
+  position: sticky;
```

Comment:
For setting the position we can use a uswds utility class, `position-sticky`

https://designsystem.digital.gov/utilities/display/#position

Reply:
Updated to use the USWDS class